ttyd/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: ttyd_web_1
+      APP_PORT: 7681
+
+  web:
+    image: tsl0922/ttyd:1.7.3
```

Comment:
We need to include the digest for the multi-arch image of ttyd to ensure that we are pulling the exact same image every time, regardless of whether the tag has been updated. See here for instructions: https://github.com/getumbrel/umbrel-apps/pull/449/files#r1146952734